### PR TITLE
softlockup: remove '-f' to avoid pkill other process unexpectedly

### PR DESCRIPTION
**Pull request by vivianQizhu**
Without the option "-f", pkill only matched against the process
name. When "-f" is set, the full command line is used. It now
kills some other process that we need to keep running during the
test so remove it.

id: 1541729, 1869584
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@yanan-fu Would you help review it? Thanks.

File: qemu/tests/cfg/softlockup.cfg [hunk 1]
```diff
@@ -4,11 +4,11 @@
...
     stress_source = stress-1.0.4.tar.gz
     stress_setup_cmd = "cd %s && tar xvf stress-1.0.4.tar.gz && cd stress-1.0.4 && ./configure && make && cd src"
-    server_setup_cmd = "%s/heartbeat_slu.py --server --threshold %s --file %s --port %s --verbose --check-drift"
-    client_setup_cmd = "%s/heartbeat_slu.py --client --address %s --file %s --port %s --interval 1"
+    server_setup_cmd = "`command -v python python3 | head -1` %s/heartbeat_slu.py --server --threshold %s --file %s --port %s --verbose --check-drift"
+    client_setup_cmd = "`command -v python python3 | head -1` %s/heartbeat_slu.py --client --address %s --file %s --port %s --interval 1"
     stress_cmd  = "cd %s && cd stress-1.0.4 && cd src && nohup ./stress -c %s > /dev/null 2>&1&"
     kill_monitor_cmd = "ps aux | grep heart | grep -v grep | awk '{print$2}' | xargs kill -9 > /dev/null 2>&1"
-    kill_stress_cmd = "pkill -f stress > /dev/null 2>&1"
+    kill_stress_cmd = "pkill stress > /dev/null 2>&1"
```
Comment (yanan-fu) [inline qemu/tests/cfg/softlockup.cfg:11]: How about `pkill -9`

Comment (vivianQizhu) [reply qemu/tests/cfg/softlockup.cfg]: @yanan-fu Updated, thanks.

**Comment by yanan-fu** [comment]
Call for test result, thanks ~

**Comment by vivianQizhu** [comment]
@yanan-fu Test does not pass, seems "heartbeat_slu.py" has other issues:

05:55:26 INFO | Running '`command -v python python3 | head -1` /var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/deps/softlockup/heartbeat_slu.py --server --threshold 10 --file /tmp/heartbeat_server.log --port 13330 --verbose --check-drift'
05:55:26 INFO | Command '`command -v python python3 | head -1` /var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/deps/softlockup/heartbeat_slu.py --server --threshold 10 --file /tmp/heartbeat_server.log --port 13330 --verbose --check-drift' finished with 0 after 0.0399477481842041s
05:55:26 DEBUG| [stderr] Traceback (most recent call last):
05:55:26 DEBUG| [stderr]   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/deps/softlockup/heartbeat_slu.py", line 215, in <module>
05:55:26 DEBUG| [stderr]     threshold, check_drift)
05:55:26 DEBUG| [stderr]   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/deps/softlockup/heartbeat_slu.py", line 52, in run_server
05:55:26 DEBUG| [stderr]     daemonize(output_file=path)
05:55:26 DEBUG| [stderr]   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/deps/softlockup/heartbeat_slu.py", line 27, in daemonize
05:55:26 DEBUG| [stderr]     output_handle = open(output_file, 'a+', 0)

**Comment by vivianQizhu** [comment]
Close for now, will update when correct all issues.

**Pull request closed without merging**